master/internal/stream/system.go

```diff
@@ -167,21 +167,21 @@ func (ps *PublisherSet) Start(ctx context.Context) error {
 	return eg.Wait()
 }
 
-func writeAll(socket *websocket.Conn, msgs []*websocket.PreparedMessage) error {
+func writeAll(socketLike WebsocketLike, msgs []interface{}) error {
```

Comment:
only if it makes the test code easier: you could make WriteAll the interface method instead of Write.

But if the test is easier to write with Write as the interface method then keep it as is.